public/static/docs/command-reference/import.md

```diff
@@ -15,17 +15,18 @@ import.
 usage: dvc import [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
-  path        Path to data within DVC repository.
+  url                   Location of DVC project or Git repository to download from
+  path                  Path to a file or directory within the project or repository
+
```

Comment:
Same comment as https://github.com/iterative/dvc.org/pull/912#discussion_r366042922.
And please remove the last (empty) new line in the code block.

Reply:
Done, thanks. Except for the blank line 😋 – will resolve this and reopen.